README.md

```diff
@@ -61,12 +61,23 @@ Run the production server from the bundled environment:
     bin/env bin/bundle exec bin/thin -p $PORT start
 
 ### Deploy with Docker
+
 To deploy PromDash with Docker, use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/).
```

Comment:
While you're editing this, could you change this to:

`...use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/) Docker image.` (added words at the end)